src/ldf-client-ui.js

```diff
@@ -547,13 +576,21 @@ require('yasgui-yasqe/dist/yasqe.css'); // Make webpack import the css as well
       if (!datasources || !datasources.length)
         return alert('Please choose a datasource to execute the query.');
 
+      // Hide map
+      mapView = false;
+      if (!mapView)
```

Comment:
If-check can probably be removed, since it's always false here.

Reply:
Done